recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ build:
   number: 3
   # TODO: windows support should be available from next version;
   #       CPU-only support still under development
-  skip: true  # [win or cuda_compiler_version == "None" or cuda_compiler_version == "10.2"]
+  skip: true  # [win or cuda_compiler_version == "None"]
```

Comment:
Dropped the CUDA 10.2 `skip` since the conda-forge minimum is now CUDA 11.2 ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1708 )